content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,143 @@
-===================================
-Connect an IoT Box to your database
-===================================
+=======================================
+Connect an IoT Box to the Odoo database
+=======================================
 
-Install the Internet of Things (IoT) App on your Odoo Database.
+Install the :menuselection:`Internet of Things (IoT)` App on the Odoo database.
 
 .. image:: connect/connect01.png
    :align: center
+   :alt: The Internet of Things (IoT) App on the Odoo database.
 
-Go in the IoT App and click on Connect on the IoT Boxes page.
+Go to the :menuselection:`IoT` App and click on :guilabel:`Connect`` on the
```

Comment:
:guilabel:\`Connect\`\` -> :guilabel:\`Connect\` (extra \`  character at the end)

Reply:
Nice find, thanks for catching this!